### DISCUSSION TITLE
Native Buttons Like Official Cloud API. #Baileys #Noweb

### DISCUSSION THREAD

**Discussion by FaheemAlvii** [devlikeapro/waha#553] [General]
Is there a way to send buttons same as Official API Does I checked and found out it is still working for some people who are using baileys.

https://github.com/WhiskeySockets/Baileys/issues/56

**Comment by devlikepro** [comment]
Hi! Yeah, likely we'll work in that direction soon :crossed_fingers: 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by ceosarzone** [comment]
any update on this?

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
